.github/workflows/verify-md.yml

```diff
@@ -0,0 +1,47 @@
+name: Markdown
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**/*.md'
+
+jobs:
+  markdown-lint:
+    runs-on: ubuntu-latest
+    name: Lint Markdown
+    steps:
+      - uses: actions/checkout@v4
+      - name: Markdown Linting
+        uses: nosborn/github-action-markdown-cli@v3
+        with:
+          files: .
+          dot: true
+          config_file: .markdownlint.yml
+          ignore_path: .markdownignore
+  verify-links:
+    name: Verify links
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Markdown verify links
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: yes
+          use-verbose-mode: no
+          config-file: .markdown-link-check.json
+  spellcheck:
+    name: Spellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Spellcheck
+        uses: rojopolis/spellcheck-github-actions@v0
+        with:
+          task_name: Markdown
+          output_file: spellcheck-output.txt
+
```

Comment:
We seem to use a lot of new "config" files that are littering the Repository. Maybe we could consider to create a new directory in `.github/etc` or `.github/config`, so that the repository looks clean and structured.

Since it looks like we can specify the path to these config-files this should be possible. Or does the actions expect the files to be at some specific location?

What do you think?

Reply:
moved GHA config files to .github/config